test/support/database.yml

```
@@ -9,10 +9,11 @@ mysql:
 
 postgresql:
   adapter: postgresql
-  host: localhost
-  port: 5432
+  host: <%= ENV.fetch('POSTGRESQL_HOST', 'localhost') %>
+  port: <%= ENV.fetch('POSTGRESQL_PORT', 5432) %>
   database: globalize_test
   username: postgres
+  password: <%= ENV['POSTGRESQL_PASSWORD'] %>
```

```suggestion
  host: <%= ENV.fetch("PGHOST", "localhost") %>
  port: <%= ENV.fetch("PGPORT", 5432) %>
  database: globalize_test
  username: postgres
<% if ENV["PGPASSWORD"] -%>
  password: <%= ENV["PGPASSWORD"] %>
<% end %>
```

Maybe we can use these vars which are documented in the postgresql docs as well: https://www.postgresql.org/docs/current/libpq-envars.html

---

Hi @parndt, thanks for answer, updated!